libs/dh/shared/ui-util/src/lib/dh-em-dash-fallback.pipe.ts

```diff
@@ -23,6 +23,7 @@ export const pipeName = 'dhEmDashFallback';
 
 @Pipe({
   name: pipeName,
```

Comment:
nitpick: now that the test is not using Spectacular anymore, the `TValue` and `pipeName` don't have be exported. The `pipeName` can directly go in the pipe's name instead.